install-prebuilt-uefi/hikey-efi-prebuilt.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Fuchsia Authors
+#
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT
+
+#set -e
+
+if [ "$#" -ne 1 ]; then
+	echo "Usage: hikey-efi-flash-image <release/debug>"
+	exit 1
+fi
+
+RELEASE=$1
+UEFI_DIR=${1:-./uefi}
+UEFI_URL="http://builds.96boards.org/snapshots/reference-platform/components/uefi-staging/14/hikey960/$RELEASE/"
```

Comment:
Your scripts uses build #14. I know so far this is the only working one. You can use that as default. But I do want you give the user freedom to choose. See, I would like something:
UEFI_URL=${BASE_URL}/${VERSION}/${PRODUCT}/${RELEASE}

Reply:
Here is a script I wrote locally. Would you please incorporate it into yours?

BASE_URL=http://builds.96boards.org/snapshots/reference-platform/components/uefi-staging
VERSION=20
PRODUCT=hikey960
RELEASE=release

while [[ $# -gt 0 ]]
do
key="$1"

case $key in
	-h)
		echo "Look the file to get help"
		echo "Use -d or -r to download debug or release version"
		echo "Use -v to specify build no."
		echo "Example: "$0" -r -v 20"
		echo "    , this downloads build #20 release version"
		exit 0
		;;
	-d|--debug)
		RELEASE=debug
		;;
	-r|--release)
		RELEASE=release
		;;
	-v|--version)
		VERSION=$2
		shift
		;;
	*)
		echo "Unknown option. Use -h for help"
		exit 0
		;;
esac
shift
done

TARGET_URL=${BASE_URL}/${VERSION}/${PRODUCT}/${RELEASE}